### DISCUSSION TITLE
Cant connect with addressbook

### DISCUSSION THREAD

**Discussion by SRinformatica-sev** [rustdesk/rustdesk-server-pro#395] [Q&A] [answered]
Hi, i´ve installed rusdesk pro on a IONOS VPS.
It´s working but a cant connect with my addressbook. When a try to log in with my user in rustdesk app i recibe an error message.

![Captura](https://github.com/user-attachments/assets/c77ad439-8849-40f8-9081-be9861d0aa08)

I have opened the ports indicated in the instructions: 21114-9 TCP, 21116 UDP...
Thanks for your help

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/wiki/FAQ#failed-to-log-in-with-custom-client-build-or-public-release-with-wrong-api-settings

**Reply by SRinformatica-sev** [reply]
Thanks!

I have deleted the port and now i recibe a different error:
![Captura2](https://github.com/user-attachments/assets/d133e0e0-79a3-4e13-a57f-ae893497a9e0)
